azure/table_azure_alert_management.go

```diff
@@ -0,0 +1,415 @@
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.Essentials.Severity"),
+			},
+			{
+				Name:        "signal_type",
+				Description: "The type of signal the alert is based on, which could be metrics, logs or activity logs. Possible values include: 'Metric', 'Log', 'Unknown'.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.Essentials.SignalType"),
+			},
+			{
+				Name:        "alert_state",
+				Description: "Alert object state, which can be modified by the user. Possible values include: 'AlertStateNew', 'AlertStateAcknowledged', 'AlertStateClosed'.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.Essentials.AlertState"),
+			},
+			{
+				Name:        "monitor_condition",
+				Description: "Can be 'Fired' or 'Resolved', which represents whether the underlying conditions have crossed the defined alert rule thresholds. Possible values include: 'Fired', 'Resolved'.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.Essentials.MonitorCondition"),
+			},
+			{
+				Name:        "target_resource",
+				Description: "Target ARM resource, on which alert got created.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.Essentials.TargetResource"),
+			},
+			{
+				Name:        "target_resource_name",
+				Description: "Name of the target ARM resource name, on which alert got created.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.Essentials.TargetResourceName"),
+			},
+			{
+				Name:        "target_resource_type",
+				Description: "Resource type of target ARM resource, on which alert got created.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.Essentials.TargetResourceType"),
+			},
+			{
+				Name:        "monitor_service",
+				Description: "Monitor service on which the rule(monitor) is set. Possible values include: 'ApplicationInsights', 'ActivityLogAdministrative', 'ActivityLogSecurity', 'ActivityLogRecommendation', 'ActivityLogPolicy', 'ActivityLogAutoscale', 'LogAnalytics', 'Nagios', 'Platform', 'SCOM', 'ServiceHealth', 'SmartDetector', 'VMInsights', 'Zabbix', 'ResourceHealth'.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.Essentials.MonitorService"),
+			},
+			{
+				Name:        "alert_rule",
+				Description: "Rule(monitor) which fired alert instance. Depending on the monitor service, this would be ARM id or name of the rule.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.Essentials.AlertRule"),
+			},
+			{
+				Name:        "source_created_id",
+				Description: "Unique Id created by monitor service for each alert instance. This could be used to track the issue at the monitor service, in case of Nagios, Zabbix, SCOM etc.",
```

Comment:
```suggestion
				Description: "Unique ID created by monitor service for each alert instance. This could be used to track the issue at the monitor service, in case of Nagios, Zabbix, SCOM, etc.",
```